### PR TITLE
Feature/39198 remove check pins once expired

### PR DESCRIPTION
**Pull request by jon-shipley**
Add function to consumption plan to delete expired checkPins every eight hours.

### CODE REVIEW

File: func-consumption/check-pin-expiry/function.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{
+  "bindings": [
+    {
+      "_ref": "sec, min, hour, day, month, dow",
```
Comment (GuyHarwood) [inline func-consumption/check-pin-expiry/function.json]: whats the `_ref` for?  is it an undocumented setting?

File: tslib/src/functions/check-pin-expiry/check-pin-expiry.service.ts [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+
+export class CheckPinExpiryService {
+  private sqlService: ISqlService
+
+  constructor (theSqlService: ISqlService) {
+    this.sqlService = theSqlService
+  }
+
+  async process () {
+    const sql = `DELETE FROM [mtc_admin].[checkPin] WHERE pinExpiresAt < GETUTCDATE()`
```
Comment (GuyHarwood) [inline tslib/src/functions/check-pin-expiry/check-pin-expiry.service.ts]: i did question why you didn't implement a dedicated data service, but this is leaner and the service is simple enough 👌

File: func-consumption/check-pin-expiry/function.json (hunk 1 above)
Comment (jon-shipley) [reply func-consumption/check-pin-expiry/function.json]: It's a comment to remind the viewer what the format of the cron expression is.   As this is JSON commenting has to be a field.

Comment (GuyHarwood) [reply func-consumption/check-pin-expiry/function.json]: good idea 💡

**Comment by GuyHarwood** [comment]
@jon-shipley conflict in tslib/package.json

**Pull request merged**